### PR TITLE
Replace all usages of container-fluid class

### PR DESCRIPTION
**Pull request by KKoukiou**
Page && PageSection compoents should be used in the react pages and the
corresponding classes in the non-react.
container-fluid class comes from bootstrap which we try to stop using.

### CODE REVIEW

**Comment by marusak** [comment]
This branch has conflicts that must be resolved

File: pkg/systemd/logs.html [hunk 1]
```diff
@@ -29,95 +29,100 @@
...
 
-  <div id="journal-entry" class="container-fluid" hidden>
-    <ol class="breadcrumb">
-      <li><a role="link" tabindex="0" id="journal-navigate-home" translate="yes">Logs</a></li>
-      <li id="journal-entry-crumb" class="active" translate="yes">Entry</li>
-    </ol>
-    <h2 id="journal-entry-heading"></h2>
-    <table class="info-table-ct panel panel-default" id="journal-entry-fields">
-    </table>
+  <div id="journal-entry" class="pf-c-page" hidden>
```
Comment (marusak) [inline pkg/systemd/logs.html:114]: Somewhere here is some problem, there is massive gap on top of a log entry.
massivegap

**Review by marusak — CHANGES_REQUESTED** [review]
Nice!
There is conflict and in logs seems there is some conflicting style.

File: pkg/systemd/logs.html (hunk 1 above)
Comment (marusak) [reply pkg/systemd/logs.html:114]: Oh and it misses breadcrumb. Can you please also write test for it? Thanks!

Comment (marusak) [reply pkg/systemd/logs.html:114]: This is the current master:
masternow

Comment (KKoukiou) [reply pkg/systemd/logs.html:114]: Fixed. Thanks for noticing!
Screen Shot 2020-10-13 at 11 16 29

Comment (KKoukiou) [reply pkg/systemd/logs.html:114]: Ups I forgot the test - will repost again.

File: pkg/systemd/logs.html [hunk 2]
```diff
@@ -29,99 +29,147 @@
...
+<<<<<<< Updated upstream
```
Comment (marusak) [inline pkg/systemd/logs.html:115]: Forgotten after rebase :/

File: pkg/systemd/logs.html [hunk 3]
```diff
@@ -29,99 +29,147 @@
...
+         </li>
+        </ol>
+     </nav>
+    </section>
+    <section class="pf-c-page__main-section pf-m-light">
+      <h2 id="journal-entry-heading"></h2>
+      <table class="info-table-ct panel panel-default" id="journal-entry-fields">
+      </table>
+    </section>
+=======
```
Comment (marusak) [inline pkg/systemd/logs.html:141]: Oh, the whole rebase is here, can you please fix? Thanks!

**Review by marusak — APPROVED** [review]
Nice, thank you!

**Pull request merged**